src/Integration/LanguageGroup.cs

```diff
@@ -1,14 +1,15 @@
 //-----------------------------------------------------------------------
-// <copyright file="RuleSetGroup.cs" company="SonarSource SA and Microsoft Corporation">
+// <copyright file="LanguageGroup.cs" company="SonarSource SA and Microsoft Corporation">
 //   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
 //   Licensed under the MIT License. See License.txt in the project root for license information.
 // </copyright>
 //-----------------------------------------------------------------------
 
 namespace SonarLint.VisualStudio.Integration
 {
-    public enum RuleSetGroup
+    public enum LanguageGroup
```

Comment:
> it can change in the middle

The Quality Profile can change in the middle of a binding??

> its too risky and unrelated change to do part of this work.

Fair enough. Perhaps we should add a technical ticket?